console/Router.php

```diff
@@ -8,6 +8,8 @@
 
 namespace lithium\console;
 
+use lithium\util\Inflector;
+
 /**
  * The `Router` class uses an instance of `lithium\console\Request`, which represents an incoming
  * command-line invocation, to parse the correct command, and sub-command(s) and parameters, which
```

Comment:
It would be great if it could be mentioned, that "foo-bar" in results is deprecated in favor of "fooBar". This will later help us remove deprecated features.